leasing/management/commands/set_group_field_permissions.py

```diff
@@ -490,15 +490,6 @@
         "due_dates_per_year": {6: "change"},
         "due_dates": {6: "change"},
         "rent_adjustments": {2: "change", 3: "change", 4: "change", 5: "change"},
-        "override_receivable_type": {
```

Comment:
Is this field permission really not needed under contractrent?

Reply:
No, because contract rent default permissions are identical to the desired permissions for this field: https://github.com/City-of-Helsinki/mvj/blob/0ae807b6d7eddd53ba19c69e511bcc68904ed9a5/leasing/management/commands/set_group_field_permissions.py#L70

Reply:
The desired permissions are briefly visible in this commit, when the field was still part of rent: 0366e81c7813bf9d4f6124eefc5b5bd788bc6eaf